synapse/rest/client/v2_alpha/account.py

```diff
@@ -65,9 +94,90 @@ def on_POST(self, request):
         if existingUid is None:
             raise SynapseError(400, "Email not found", Codes.THREEPID_NOT_FOUND)
 
-        ret = yield self.identity_handler.requestEmailToken(**body)
+        if self.config.email_enable_password_reset_from_is:
+            if 'id_server' not in body:
+                raise SynapseError(400, "Missing 'id_server' param in body")
```

Comment:
It would make sense to me for the server to be able to default this?

Reply:
It's currently a required field in [the CS spec](https://matrix.org/docs/spec/client_server/unstable.html#post-matrix-client-r0-account-password-email-requesttoken) though.